Installing-and-Configuring/OS/Generic.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: default
+title: Generic
+permalink: /OS/Generic/
+parent: Operating Systems
+grand_parent: Installing and configuring
+nav_order: 1
+---
+
+<!-- markdownlint-disable MD033 -->
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+<!-- markdownlint-enable MD033 -->
+
+Generic OS Installation
+===
+
+Insert OS installation media into one of the USB3 ports (blue on Thinkpads).
+ Turn the system on and navigate to the USB boot in the graphical heads menu. 
+
+Main Menu => Boot Options => USB Boot
+
+From here you will be presented with a menu of the boot options on the USB media.  Choose the one you wish to use and Press enter.  It is your responsibility to verify the integrity of the media before using it.  Check your downloads against the signatures provided by vendors.
+
+
+Securely Booting Installation Media
+----
+
+Heads also supports booting directly from verified ISOs on a standard disk partition.  In this scenario heads will check the ISO against a corresponding signature.  The ISO must be signed by a valid key for the boot process to succeed.  
+
+If not using one of the included Operating Systems--where the vendor public key is [stored in the heads ROM](https://github.com/osresearch/heads/tree/master/initrd/etc/distro/keys)--you must verify the media immediately after download and sign it with your own key to establish a chain of trust.
+
+To verify an ISO with your key, create a partition with a Heads compatible filesystem on a USB storage device and copy the ISO image to it.  The most common filesystems are: FAT and ext. The layout might look like this:
+
+```shell
+/Qubes-R4.0-x86_64.iso
+/Qubes-R4.0-x86_64.iso.asc
+/Fedora-Workstation-Live-x86_64-27-1.6.iso
+/Fedora-Workstation-Live-x86_64-27-1.6.iso.sig
+/tails-amd64-3.7.iso
+/tails-amd64-3.7.iso.sig
+```
+
+Sign the ISO(s) with your own key:
+
+```shell
+gpg --output <iso_name>.sig --detach-sig <iso_name>
+```
+
+### Distro Specific ISO Boot
+
+Some distros require additional options for a successful boot directly from ISO.  See [Boot config files](#boot-config-files) for more information.
+1. Boot from USB by either running `usb-scan` or reboot into USB boot mode (hit
```

Comment:
Some distros require additional options for a successful boot directly from ISO.  See [Boot config files](#boot-config-files) for more information if your OS is not booting selecting default boot options from GUI.

Reply:
incorporated. push in a few minutes.